sources/packages/backend/apps/workers/src/zeebe/zeebe-transport-strategy.ts

```diff
@@ -72,7 +71,9 @@ export class ZeebeTransportStrategy
    */
   private async workerHandlerWrapper(
     jobHandler: MessageHandler,
-    job: Readonly<ZeebeJob<IInputVariables, ICustomHeaders, IOutputVariables>>,
+    job: Readonly<
+      ZeebeJob<{ [x: string]: unknown }, ICustomHeaders, IOutputVariables>
```

Comment:
Is there any implication if we just use `ZeebeJob<unknown, ICustomHeaders, IOutputVariables>` ?

Reply:
`unknown` can be used but will be less specific in this case.
For this one, the type suggested was used. Since it is more specific than `unknow` I would prefer to leave as it is.
![image](https://github.com/bcgov/SIMS/assets/61259237/3b85c002-0ea2-41bb-aacb-10433ff64308)